### PR TITLE
zd3873417: Update pay-js-commons

### PR DESCRIPTION
**Pull request by oswaldquek**
## WHAT
_A brief description of the pull request:_

## HOW 
_Steps to test or reproduce:_

### CODE REVIEW

**Comment by stephencdaly** [comment]
probably need to commit package-lock.json after running `npm install`

File: package.json [hunk 1]
```diff
@@ -122,7 +122,7 @@
     "chai-string": "^1.4.0",
     "cheerio": "^1.0.0-rc.3",
     "chokidar-cli": "latest",
-    "cypress": "3.1.5",
+    "cypress": "^3.8.0",
```
Comment (stephencdaly) [inline package.json]: I think we prefer to keep dependencies anchored to a specific version nowadays as dependabot will upgrade versions. So remove the `^` I think

Comment (oswaldquek) [reply package.json]: This happened automatically as a result of running `npm audit fix` but sure I'll remove it.

Comment (stephencdaly) [reply package.json]: looks like browser tests have failed, so we probably can't upgrade

**Comment by stephencdaly** [comment]
I think we use a docker image of cypress on CI and the version probably has to match, so don't think upgrading cypress works without some more effort.

**Pull request merged**